wdl-grammar/src/v1/lint/double_quotes.rs

```diff
@@ -0,0 +1,148 @@
+//! All quotes should be double quotes.
+
+use std::collections::VecDeque;
+
+use nonempty::NonEmpty;
+use pest::iterators::Pair;
+use wdl_core::concern::code;
+use wdl_core::concern::lint;
+use wdl_core::concern::lint::Rule;
+use wdl_core::concern::lint::TagSet;
+use wdl_core::concern::Code;
+use wdl_core::file::Location;
+use wdl_core::Version;
+
+use crate::v1;
+
+///
```

Comment:
Does this need a comment?

Reply:
Yes, forgot this one. Strangely none of the steps described in the README picked it up but the lint here did.

Will add.